doc/changelog.rst

```diff
@@ -3,13 +3,21 @@ Changelog
 
 Changes in Version 4.8.0
 -------------------------
+
 .. warning:: PyMongo 4.8 drops support for Python 3.7 and PyPy 3.8: Python 3.8+ or PyPy 3.9+ is now required.
 
 PyMongo 4.8 brings a number of improvements including:
+
 - The handshake metadata for "os.name" on Windows has been simplified to "Windows" to improve import time.
 - The repr of ``bson.binary.Binary`` is now redacted when the subtype is SENSITIVE_SUBTYPE(8).
 - A new asynchronous API with full asyncio support.
 
+Unavoidable breaking changes
+............................
+
+- Since we are now using ``hatch`` as our build backend, we no longer have a ``setup.py`` file
+  and require installation using ``pip``.
```

Comment:
This should be in 4.9 not 4.8, unless we are planning to backport PYTHON-4451.

Reply:
It will be 4.8, PR is already open: https://github.com/mongodb/mongo-python-driver/pull/1663